package.json

```diff
@@ -56,6 +52,7 @@
   "devDependencies": {
     "@babel/core": "7.16.0",
     "@babel/parser": "7.16.0",
+    "@mongodb-js/bump-monorepo-packages": "^0.1.1",
```

Comment:
Where does this live? https://www.npmjs.com/package/@mongodb-js/bump-monorepo-packages says it’s in this repository but it doesn’t seem to actually be here. (I’m asking because I’m wondering whether this package takes into account different semver bumps per package, depending on what types of commit have touched it.)

Reply:
Seems like package.json has a wrong repo mentioned. Here's the package https://github.com/mongodb-js/devtools-shared/tree/main/packages/bump-monorepo-packages